daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_expressions/expressions.tdml

```diff
@@ -2440,38 +2440,47 @@
      Purpose: This test demonstrates that checkConstraints cannot reference an inputValueCalc-ed element because the value is being computed by inputValueCalc
 -->
 
-  <tdml:parserTestCase name="dfdlCheckConstraints"
-    root="e1" model="checkConstraints" description="DFDL-23-135R">
+  <tdml:parserTestCase name="dfdlCheckConstraintsFails"
+                       root="e1" model="checkConstraints" description="DFDL-23-135R">
     <tdml:document>6</tdml:document>
     <tdml:errors>
       <tdml:error>Schema Definition Error</tdml:error>
-      <tdml:error>Placeholder</tdml:error>
+      <tdml:error>you cannot use checkConstraints(.)</tdml:error>
     </tdml:errors>
   </tdml:parserTestCase>
 
-  <tdml:parserTestCase name="checkConstraintsComplexTypeFails"
-    root="e7" model="checkConstraints" description="DFDL-23-135R">
-    <tdml:document>s1s2</tdml:document>
-    <tdml:errors>
-      <tdml:error>Schema Definition Error</tdml:error>
-      <tdml:error>checkConstraints</tdml:error>
-      <tdml:error>AnyAtomic</tdml:error>
-    </tdml:errors>
-  </tdml:parserTestCase>
+<!--  <tdml:parserTestCase name="dfdlCheckConstraints"-->
+<!--    root="e1" model="checkConstraints" description="DFDL-23-135R">-->
+<!--    <tdml:document>6</tdml:document>-->
+<!--    <tdml:errors>-->
+<!--      <tdml:error>Schema Definition Error</tdml:error>-->
+<!--      <tdml:error>Placeholder</tdml:error>-->
+<!--    </tdml:errors>-->
+<!--  </tdml:parserTestCase>-->
+
+<!--  <tdml:parserTestCase name="checkConstraintsComplexTypeFails"-->
+<!--    root="e7" model="checkConstraints" description="DFDL-23-135R">-->
+<!--    <tdml:document>s1s2</tdml:document>-->
+<!--    <tdml:errors>-->
+<!--      <tdml:error>Schema Definition Error</tdml:error>-->
+<!--      <tdml:error>checkConstraints</tdml:error>-->
+<!--      <tdml:error>AnyAtomic</tdml:error>-->
+<!--    </tdml:errors>-->
+<!--  </tdml:parserTestCase>-->
 
 <!-- 
      Test Name: dfdlCheckConstraints2
      Schema: checkConstraints
      Purpose: This test demonstrates that checkConstraints cannot reference an inputValueCalc-ed element because the value is being computed by inputValueCalc
 -->
-  <tdml:parserTestCase name="dfdlCheckConstraints2" validation="on"
-    root="e8" model="checkConstraints" description="DFDL-23-135R">
-    <tdml:document></tdml:document>
-    <tdml:errors>
-      <tdml:error>Schema Definition Error</tdml:error>
-      <tdml:error>Placeholder</tdml:error>
-    </tdml:errors>
-  </tdml:parserTestCase>
+<!--  <tdml:parserTestCase name="dfdlCheckConstraints2" validation="on"-->
+<!--    root="e8" model="checkConstraints" description="DFDL-23-135R">-->
+<!--    <tdml:document></tdml:document>-->
+<!--    <tdml:errors>-->
+<!--      <tdml:error>Schema Definition Error</tdml:error>-->
+<!--      <tdml:error>Placeholder</tdml:error>-->
+<!--    </tdml:errors>-->
+<!--  </tdml:parserTestCase>-->
```

Comment:
Why comment out these tests? We usually keep TDML content  uncommented and keep the tests in the scala file as commented if things don't work.